blocks/accordion/accordion.css

```diff
@@ -0,0 +1,207 @@
+:root {
+    --accordion-row-border-color: #b3b7c9;
+    --accordion-row-title-color: #172660;
+    --expandable-button-color: #e81b75 ;
+}
+
+.accordion .accordion-row {
+    margin-bottom: 13px;
+}
+
+.accordion .accordion-row .accordion-row-content {
+    border: 2px solid var(--accordion-row-border-color);
+    padding: 10px 12px;
+    position: relative;
+    min-height: 60px;
+}
+
+.accordion .accordion-row .accordion-row-content .accordion-row-header {
+    display: flex;
+    align-items: center;
+    height: 60px;
+    min-height: 60px;
+}
+
+.accordion .accordion-row .accordion-row-content > p {
+    display: none;
+}
+
+.accordion .accordion-row.expanded .accordion-row-content > p {
+    display: block;
+}
+
+.accordion .accordion-row .accordion-row-content .accordion-row-header img {
+    min-width: 88px;
+    max-width: 88px;
+    margin-right: 20px;
+}
+
+.accordion .accordion-row .accordion-row-content .accordion-row-header .accordion-title-header-container {
+    display: flex;
+    flex-direction: column;
+    flex: 2;
+}
+
+.accordion .accordion-row .accordion-row-content .accordion-row-header .accordion-title-header-container h2 {
+    font-size: var(--heading-font-size-m);
+    color: var(--accordion-row-title-color);
+    font-weight: 500;
+    margin: 0;
+    padding-right: 40px;
+}
+
+.accordion .accordion-row.expanded .accordion-row-content p {
+    font-size: var(--body-font-size-s);
+}
+
+.accordion .accordion-row .accordion-row-header picture {
+    min-width: 88px;
+    min-height: 58px;
+    flex: 1;
+}
+
+.accordion .accordion-row .accordion-row-content .accordion-row-header .accordion-title-header-container p {
+    display: none;
+}
+
+.accordion .accordion-row.expanded .accordion-row-content p.item-with-picture-before {
+    display: flex;
+    align-items: center;
+}
+
+.accordion .accordion-row.expanded .accordion-row-content p.item-with-picture-before div {
+    padding-left: 12px;
+}
+
+.accordion .accordion-row.expanded .accordion-row-content p.item-with-picture-before picture img {
+    width: 35px;
+    padding-right: 12px;
+    max-width: unset;
+}
+
+.accordion .accordion-row .accordion-row-content .expandable-button {
+    position: absolute;
```

Comment:
can you add `cursor: pointer` to this button?

Reply:
fixed the issue, had to add `pointer-events: auto` because it was not showing the pointer